ls1intum/Artemis
# `General`: Add exam mode and feature overview docs for exam checklist and grading keys

### Checklist
#### General



### Motivation and Context


The exam mode documentation is missing the following two important concepts: Checklist and Grade key.

### Description

The exam checklist and grade key documentation is added to Exam Mode Guides (Instructors' Guide and Students' Guide) and also Feature Overview pages for Instructors and Students.
### Steps for Testing


1. Check docs /user/exams/instructors_guide/#exam-checklist
2. Check docs /user/exams/instructors_guide/#grading-key
3. Check docs /user/exams/students_guide/#grades
4. Check `Follow Checklist` feature page at /features/instructors
5. Check `Create Grading Keys` feature page at /features/instructors
6. Check `View Grading Keys` feature page at /features/students

### Review Progress





#### Code Review
